lib/nimbusRecipe.ts

```diff
@@ -271,7 +292,8 @@ export class NimbusRecipe implements NimbusRecipeType {
       experimenterLink: `https://experimenter.services.mozilla.com/nimbus/${this._rawRecipe.slug}`,
       userFacingName: this._rawRecipe.userFacingName,
       nimbusExperiment: this._rawRecipe,
-      branches: this.getBranchInfos(),
+      branches: branchInfos,
+      isMicrosurvey: isMicrosurvey,
```

Comment:
How about renaming this to`hasMicrosurvey` here and throughout the patch, since the survey may only be a part of the experiment (like the current review checker experiments).